src/golang/lib/models/shared/service.go

```diff
@@ -37,6 +37,43 @@ const (
 	DemoDbIntegrationName = "aqueduct_demo"
 )
 
+var relationalDatabaseIntegrations map[Service]bool = map[Service]bool{
+	Postgres:     true,
+	Snowflake:    true,
+	MySql:        true,
+	Redshift:     true,
+	MariaDb:      true,
+	SqlServer:    true,
+	BigQuery:     true,
+	AqueductDemo: true,
+	Sqlite:       true,
+	Athena:       true,
+	MongoDB:      true,
+}
+
+var computeIntegrations map[Service]bool = map[Service]bool{
+	Airflow:    true,
+	Lambda:     true,
+	Conda:      true,
+	Databricks: true,
+	Kubernetes: true,
+	Spark:      true,
+}
+
+// ServiceToEngineConfigIntegrationIDField contains
+// all services with `integration_id` in its 'engine_config' field.
+// This is used in SQL queries to retrieve engine configs (workflow or operator)
+// based on integration ID.
+//
+// The key should be the service type, and value should be the json tag
+// for the corresponding field that contains the integration ID.
+var ServiceToEngineConfigIntegrationIDField map[Service]string = map[Service]string{
```

Comment:
I think calling it `ServiceToEngineConfigField` is more accurate, because the mapping value is the engine config json tag, not the integration id.